src/transformState.js

```diff
@@ -4,8 +4,25 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  actions.forEach((el) => {
```

Comment:
Названия для всех переменных должны быть значимыми и понятными (в actions перебираем action):
```suggestion
  actions.forEach((action) => {
```